ingresspipe/utils/schema_utils.py

```diff
@@ -86,7 +89,7 @@ def load_schema_into_networkx(schema):
                 else:
                     node["required"] = False
 
-
+            # not sure if this is required?
```

Comment:
Let's keep it in for now; in the future we might want to handle validation rules in a different way. Or @sujaypatil96 do you think that codepath is not used?

Reply:
Yup, I feel the `"validationRules"` handler is not required. I can illustrate/demo why during our code review call.